### PR TITLE
Add columns

### LINKED ISSUE #25: Feature request: add store view to the admin listings for landing pages and overview pages
From the admin listings it is currently not possible to see for which store view a landing page or overview page has been created.

For instance, the admin listing for the landing pages only shows the `id`, `active`, `name` and `url_path` properties of a landing page. Whenever there are two landing page with the same `name` and `url_path` for different store view it is impossible to distinguish which is which. In the following example, it can be seen in the admin listing that are two landing pages for URL path `foo/bar`, but it cannot be seen that one of them is tied to store view 1, and the other to store view 2:

admin_landing_page_listing

So I would propose to add an extra "Store View" column to the listings to show the `store_ids` property of landing pages and overview pages.

BTW As a workaround, I tried putting the store view in the `name`. That didn't work, because the name is shown in the frontend as part of the breadcrumb.

### PR DESCRIPTION
**Pull request by ah-net**
Add extra columns to attribuut landingpage overview and overview page

fixes #25

### CODE REVIEW

File: src/Ui/Component/Listing/Column/OverviewPageActions.php [hunk 1]
```diff
@@ -16,6 +17,8 @@ class OverviewPageActions extends Column
     const URL_PATH_EDIT = 'emico_attributelanding/overviewpage/edit';
     const URL_PATH_DELETE = 'emico_attributelanding/overviewpage/delete';
 
+    public $storeRepository;
```
Comment (Hnto) [inline src/Ui/Component/Listing/Column/OverviewPageActions.php:20]: Please add the type

File: src/Ui/Component/Listing/Column/OverviewPageActions.php [hunk 2]
```diff
@@ -70,6 +75,20 @@ public function prepareDataSource(array $dataSource)
                         ]
                     ];
                 }
+
+                if (!empty($item['store_ids'])) {
+                    $store_ids = explode(',', $item['store_ids']);
```
Comment (Hnto) [inline src/Ui/Component/Listing/Column/OverviewPageActions.php:80]: Please add a check on the store_ids property

File: src/Ui/Component/Listing/Column/PageActions.php [hunk 3]
```diff
@@ -10,6 +10,16 @@ class PageActions extends \Magento\Ui\Component\Listing\Columns\Column
      */
     protected $urlBuilder;
 
+    /**
+     * @var \Magento\Catalog\Model\CategoryRepository
+     */
+    public $categoryRepostory;
```
Comment (Hnto) [inline src/Ui/Component/Listing/Column/PageActions.php:16]: Add type

File: src/Ui/Component/Listing/Column/PageActions.php [hunk 4]
```diff
@@ -10,6 +10,16 @@ class PageActions extends \Magento\Ui\Component\Listing\Columns\Column
...
+
+    /**
+     * @var \Magento\Store\Model\StoreRepository
+     */
+    public $storeRepository;
```
Comment (Hnto) [inline src/Ui/Component/Listing/Column/PageActions.php:21]: add type

**Review by Hnto — CHANGES_REQUESTED** [review]

File: src/Ui/Component/Listing/Column/OverviewPageActions.php (hunk 2 above)
Comment (ah-net) [reply src/Ui/Component/Listing/Column/OverviewPageActions.php:80]: Issue is resolved. Added check on line below this statement

File: src/Ui/Component/Listing/Column/OverviewPageActions.php [hunk 5]
```diff
@@ -16,6 +17,8 @@ class OverviewPageActions extends Column
     const URL_PATH_EDIT = 'emico_attributelanding/overviewpage/edit';
     const URL_PATH_DELETE = 'emico_attributelanding/overviewpage/delete';
 
+    public StoreRepository $storeRepository;
```
Comment (Hnto) [inline src/Ui/Component/Listing/Column/OverviewPageActions.php:20]: Please use StoreRepositoryInterface instead of the model directly

**Pull request merged**